dist/setup-appliance.sh

```diff
@@ -580,6 +591,42 @@ EOF
 
 }
 
+function prepare_os_settings {
+  . /etc/os-release
+  for d in $ID_LIKE;do
```

Comment:
`ID_LIKE` doesn't exist in Fedora's `os-release` file. You'll need to concatenate `$ID` and `$ID_LIKE` into a single array first.

Reply:
In my test installation it is in there.

Taken from my test installation: 

```
[kanku@obs-server ~]$ cat /etc/os-release 
NAME=Generic
VERSION="31 (Generic)"
ID=generic
ID_LIKE=fedora
...
[kanku@obs-server ~]$ rpm -qf /etc/os-release
generic-release-common-31-2.noarch
[kanku@obs-server ~]$ rpm -qi generic-release-common-31-2.noarch
Name        : generic-release-common
...
Build Host  : buildhw-10.phx2.fedoraproject.org
Packager    : Fedora Project
Vendor      : Fedora Project
...
[kanku@obs-server ~]$

```

Reply:
`generic-release` has it, but `fedora-release` will not.

Reply:
For example, from my computer:

```
[neal@yu-ohki ~]$ cat /etc/os-release
NAME=Fedora
VERSION="31 (KDE Plasma)"
ID=fedora
VERSION_ID=31
VERSION_CODENAME=""
PLATFORM_ID="platform:f31"
PRETTY_NAME="Fedora 31 (KDE Plasma)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:31"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f31/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=31
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=31
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="KDE Plasma"
VARIANT_ID=kde
```